core/src/test/java/org/apache/accumulo/core/iterators/FirstEntryInRowIteratorTest.java

```diff
@@ -53,30 +54,33 @@ private static long process(TreeMap<Key,Value> sourceMap, TreeMap<Key,Value> res
   public void test() throws IOException {
     TreeMap<Key,Value> sourceMap = new TreeMap<>();
     Value emptyValue = new Value("".getBytes());
+    IteratorSetting iteratorSetting = new IteratorSetting(1, FirstEntryInRowIterator.class);
+    FirstEntryInRowIterator.setNumScansBeforeSeek(iteratorSetting, 10);
```

Comment:
Will the test fail if this is not set to properly?  If not, could call iteratorSetting.getOptions() after this and check if it was really set.